src/app/components/Pagination/Pagination.jsx

```diff
@@ -27,10 +27,11 @@ class Pagination extends React.Component {
     const intPage = parseInt(page, 10);
     const pageNum = type === 'Next' ? intPage + 1 : intPage - 1;
     const svg = type === 'Next' ? <RightArrowIcon /> : <LeftArrowIcon />;
+    const url = this.props.createAPIQuery({ page: pageNum });
```

Comment:
Can you also debug when you do code reviews to make sure you're right, that'll help in me having to go back and check myself.